script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -3100,12 +3106,12 @@ Shiki. Although we have spent an entire year together, I did not regret any of i
 [sha:3c41230d4544ff295ef693f346a728580cd7f1da]{
 -- Page 183, Offset 41950.
 -- それはカリウス氏にとって特上の応援だったのだろう。
-I wonder if that's his way of showing his heartfelt support.
+I suppose that was his most heartfelt way of showing support.
```

Comment:
Hmmm, they're both not great, honestly.
>I suppose that was the most heartfelt way he could show his support for me.

Reply:
Hmm, the emphasis feels ever so slightly off to me - like "that's the most heartfelt way in general he could do it" (not anchored to _by Karius's standards_), rather than "For Karius, that constitutes a ringing endorsement".
> Coming from Karius, that constitutes a show of heartfelt support.